lib/routes/follow/subscriptions.ts

```diff
@@ -0,0 +1,38 @@
+import type { Data, Route } from '@/types';
+import { Context } from 'hono';
+import ofetch from '@/utils/ofetch';
+import type { FollowResponse, Profile, Subscription } from './types';
+
+export const route: Route = {
+    name: '订阅列表',
```

Comment:
Could you change it to `User subscriptions`? And accordingly update the title returned below.

Reply:
Okay, the title is updated now.